### ISSUE TITLE
Ads Display

### ISSUE THREAD

**Issue by YrOnSai** [electronoob/agarmods#104]
Ads are not scaling with changes to the size of website.

**Comment by electronoob** [comment]
is this still an issue?

They scale when the page is refreshed though I believe?

**Comment by Mevin1** [comment]
What advertisements? The menu should be at a fixed width, and wouldn't rescale on window resize, so the advertisement shouldn't either.

**Comment by YrOnSai** [comment]
I think so, but the ads should scale like the video and be responsive.

**Comment by asweiler** [comment]
Yeah it seems to be the case with me too even to this day. Often I don't even see ads with mods and without mods I do see them which I don't understand. No ad-blocker or anything really. I can make a screenshot if asked.
